text/chapter3.md

```diff
@@ -560,20 +604,60 @@ apply f x = f x
 infixr 0 apply as $
 ```
 
-So `apply` takes a function and a value and applies the function to the value. The `infixr` keyword is used to define `($)` as an alias for `apply`.
+So `apply` takes a function and a value and applies the function to the value. The `infixr` keyword is used to define `$` as an alias for `apply`.
 
 But why would we want to use `$` instead of regular function application? The reason is that `$` is a right-associative, low precedence operator. This means that `$` allows us to remove sets of parentheses for deeply-nested applications.
 
 For example, the following nested function application, which finds the street in the address of an employee's boss:
 
 ```haskell
-street (address (boss employee))
+_.street (_.address (_.boss employee))
 ```
 
 becomes (arguably) easier to read when expressed using `$`:
 
 ```haskell
-street $ address $ boss employee
+_.street $ _.address $ _.boss employee
+```
+
+Note that neither of the above examples is idiomatic PureScript. Real-world code is more likely to express this as:
+```haskell
+(boss employee).address.street
+```
+or
+```haskell
+_.boss.address.street employee
+```
+
+There are situations where putting a prefix function in an infix position as an operator leads to more readable code. One example is the `mod` function:
+```text
+> mod 8 3
+2
+```
+This is fine, but doesn't line up with common usage. Wrapping a prefix function in backticks (\`) lets you use a prefix function in infix position as an operator, e.g.,
+```text
+> 8 `mod` 3
+2
+```
+Likewise, wrapping an operator in parentheses lets you use it as a function in prefix position:
+```text
+> 8 + 3
+11
+
+> (+) 8 3
+11
+```
+This allows for compact definitions of curried (or partially applied) functions based on infix operator functions, such as the `add2` function below:
+```text
+> add2 = (+) 2
+> add2 4
+6
+```
+Alternatively, operators can be partially applied by surrounding them with parentheses and using `_` as an operand:
```

Comment:
```suggestion
Alternatively, operators can be partially applied by surrounding them with parentheses and using `_` as an operand in an [operator section](https://github.com/purescript/documentation/blob/master/language/Syntax.md#operator-sections):
```
The content at that link is a bit lite on all the operator sections rules. Added some more details recently in https://github.com/purescript/documentation/pull/369 . Not sure if any of that should be moved into the language spec.